src/libxtp/orbitals.cc

```diff
@@ -601,6 +605,10 @@ void Orbitals::ReadFromCpt(CheckpointReader r) {
   r(_BSE_triplet, "BSE_triplet");
 
   r(_use_Hqp_offdiag, "use_Hqp_offdiag");
+
+  r(_BSE_singlet_energies_dynamic, "BSE_singlet_dynamic");
+
+  r(_BSE_triplet_energies_dynamic, "BSE_triplet_dynamic");
```

Comment:
this breaks backwards compatability of the orbitals file. If you do not want that, use the version variable like here:

https://github.com/votca/xtp/blob/56d69efde879a8ffd41f1a4134b3411d7492d3e1/src/libxtp/orbitals.cc#L579

Reply:
How exactly are those versions defined now?

Reply:
At the moment it is at version 1, so far no version 2 is used, so you can increment to version 2

Reply:
add a comment what the version changed.

Reply:
ok, I think I did what you suggested...